corehq/apps/sms/views.py

```diff
@@ -1348,7 +1353,9 @@ def column_names(self):
     def page_context(self):
         context = self.pagination_context
         context.update({
-            'initiate_new_form': InitiateAddSMSBackendForm(user=self.request.couch_user),
+            'initiate_new_form': InitiateAddSMSBackendForm(
+                user=self.request.couch_user
```

Comment:
Not necessary to pass in the `domain` here, since this view is part of admin interface.